perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -477,14 +493,18 @@ def _CliBasedTests(output_results, metadata, vm, iteration_count,
                                 'iterations.')
 
   # Report various percentiles.
+  metrics_prefix = ''
+  if FLAGS.cli_size != 'normal':
+    metrics_prefix = '%s ' % FLAGS.cli_size
```

Comment:
Seems like you want to keep the default results format compatible with old ones. I suggest adding a field in metadata cli_test_size: normal/large, which is more consistent. Up to you.